### PR TITLE
Remove ropsten

### PR DESCRIPTION
**Pull request by MarekM25**
Resolves https://github.com/NethermindEth/nethermind/issues/5331

## Changes

-  Remove ropsten

#### What types of changes does your code introduce?

- [ ] Bugfix (a non-breaking change that fixes an issue)
- [ ] New feature (a non-breaking change that adds functionality)
- [ ] Breaking change (a change that causes existing functionality not to work as expected)
- [ ] Optimization
- [x] Refactoring
- [ ] Documentation update
- [ ] Build-related changes
- [ ] Other: _Description_

#### Requires testing

- [ ] Yes
- [x] No

#### If yes, did you write tests?

- [ ] Yes
- [ ] No

#### Notes on testing

_Optional. Remove if not applicable._

#### Requires documentation update

- [x] Yes
- [ ] No

Docs in progress by @marcindsobczak

### CODE REVIEW

File: src/Nethermind/Nethermind.GitBook/SampleConfigGenerator.cs [hunk 1]
```diff
@@ -22,7 +22,7 @@ public void Generate()
             string docsDir = DocsDirFinder.FindDocsDir();
             string runnerDir = DocsDirFinder.FindRunnerDir();
             string moduleName = "sample-configuration";
-            string[] configs = { "mainnet.cfg", "goerli.cfg", "rinkeby.cfg", "ropsten.cfg" };
+            string[] configs = { "mainnet.cfg", "goerli.cfg", "seoplia.cfg" };
```
Comment (FalcoXYZ) [inline src/Nethermind/Nethermind.GitBook/SampleConfigGenerator.cs:25]: typo here

**Review by FalcoXYZ — CHANGES_REQUESTED** [review]
Besides the typo, I think the rest is OK from devops perspective :)

File: src/Nethermind/Nethermind.GitBook/SampleConfigGenerator.cs (hunk 1 above)
Comment (MarekM25) [reply src/Nethermind/Nethermind.GitBook/SampleConfigGenerator.cs]: Good catch. Thank you @FalcoXYZ !

**Review by FalcoXYZ — APPROVED** [review]
LGTM, maybe wait for someone from @NethermindEth/core to review as well :)

**Review by rubo — COMMENTED** [review]
I see we have a lot "Ropsten" mentions across the code:

- [Nethermind.PerfTest/Program.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.PerfTest/Program.cs#L257) will fail
- [Nethermind.Blockchain.Test/chainspec.json](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Blockchain.Test/chainspec.json), name
- [VirtualMachine.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Evm/VirtualMachine.cs), some comments
- [ForkInfoTests.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Network.Test/ForkInfoTests.cs#L244), some comments and possibly test cases

**Comment by MarekM25** [comment]
> I see we have a lot "Ropsten" mentions across the code:
> 
>     * [Nethermind.PerfTest/Program.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.PerfTest/Program.cs#L257) will fail
> 
>     * [Nethermind.Blockchain.Test/chainspec.json](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Blockchain.Test/chainspec.json), name
> 
>     * [VirtualMachine.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Evm/VirtualMachine.cs), some comments
> 
>     * [ForkInfoTests.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Network.Test/ForkInfoTests.cs#L244), some comments and possibly test cases
> 
>     * [BlockchainIds.cs](https://github.com/NethermindEth/nethermind/blob/remove_ropsten/src/Nethermind/Nethermind.Core/BlockchainIds.cs#L11)

Thx @rubo I did a grep search but somehow missed it. Anyway, thank you.
About VirtualMachine I had seen this comment and I decided to not change it.
About Perf.Program I doubt anyway is using it, but would be great to think about this tool. Updated to goerli for now.

**Review by rubo — APPROVED** [review]

**Pull request merged**